pyproject.toml

```diff
@@ -82,7 +74,11 @@ exclude = '''
       | cassandra
       | config.py
       | dbapi
-      | grpc
+      | grpc/
+      (
+        hello_pb2_grpc.py
+        | hello_pb2.py
```

Comment:
I know these are generated for us, but any reason not to format them?

Reply:
My thinking was that there's no guarantee that these will be generated in a `black`-compliant format.

Reply:
They probably won't be, but that doesn't mean we cannot run `black` on them before we commit them to the repo.

e.g. if we ever need to update them, re-generate, run black, then `git add`.

This is probably what I'd prefer if they were files existing inside of `ddtrace/` anyways.

What do you think?

Reply:
I have nothing against it in principle; it's perhaps not ideal for my normal workflows since I rely on "format on save" rather than a manual invocation of `black`. I'll format those two sources, update the `pyproject.toml` file and update the PR 🙂

Reply:
For context, these files have only been touched like 3-4 times in the past 2-3 years, and most of them are for flake8 formatting changes.